### PR TITLE
Fix all PEP8 non-compatible code for Regsiter

### PR DESCRIPTION
**Pull request by mabuaisha**

### CODE REVIEW

**Comment by mabuaisha** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;4Ekf9ou8&#x2F;505-1-use-git-pre-commit-hook-library-to-fix-all-pep8-non-compatible-code-banana"> (1) Use git-pre-commit-hook library to fix all PEP8 non-compatible code [Banana]</a>

File: registration/tests/test_views.py [hunk 1]
```diff
@@ -123,10 +122,10 @@ def test_reg_source_session(self):
...
 
         for key in (
-                SESSION_EMAIL, SESSION_STATE, SESSION_STATE_NAME, SESSION_POSTAL_CODE, SESSION_REGISTRATION_CONFIGURATION,
-                SESSION_ACCEPTS_REGISTRATION, SESSION_REDIRECT_URL, SESSION_REGISTRATION_UPDATE, SESSION_COBRAND_COMPANY_NAME,
-                SESSION_COBRAND_COMPANY_LOGO, SESSION_COBRAND_ACTIVE, SESSION_COBRAND_ID, SESSION_WIDGET_HOST_URL,
-                SESSION_WIDGET_ID,):
+            SESSION_EMAIL, SESSION_STATE, SESSION_STATE_NAME, SESSION_POSTAL_CODE, SESSION_REGISTRATION_CONFIGURATION,
+            SESSION_ACCEPTS_REGISTRATION, SESSION_REDIRECT_URL, SESSION_REGISTRATION_UPDATE, SESSION_COBRAND_COMPANY_NAME,
+            SESSION_COBRAND_COMPANY_LOGO, SESSION_COBRAND_ACTIVE, SESSION_COBRAND_ID,
+                SESSION_WIDGET_HOST_URL, SESSION_WIDGET_ID,):
```
Comment (waseem-omar) [inline registration/tests/test_views.py]: @mabuaisha Shouldn't this be on the indentation a the above line? Or maybe you can add it to the same line after SESSION_COBRAND_ID

File: registration/tests/test_views.py [hunk 2]
```diff
@@ -143,12 +142,12 @@ def test_session_cleaning_on_start(self):
...
 
-        r = self.client.get('/', follow=False)
+        self.client.get('/', follow=False)
         session = self.client.session
         for key in (
             SESSION_EMAIL, SESSION_STATE, SESSION_STATE_NAME, SESSION_POSTAL_CODE, SESSION_REGISTRATION_CONFIGURATION,
             SESSION_ACCEPTS_REGISTRATION, SESSION_REDIRECT_URL, SESSION_REGISTRATION_UPDATE, SESSION_REG_SOURCE, SESSION_VARIANT_ID,
             SESSION_COBRAND_COMPANY_NAME, SESSION_COBRAND_COMPANY_LOGO, SESSION_COBRAND_ACTIVE, SESSION_COBRAND_ID,
-            SESSION_WIDGET_HOST_URL, SESSION_WIDGET_ID,):
+                SESSION_WIDGET_HOST_URL, SESSION_WIDGET_ID,):
```
Comment (waseem-omar) [inline registration/tests/test_views.py]: @mabuaisha Shouldn't this be on the same indentation as the upper line?

File: registration/tests/test_views.py (hunk 1 above)
Comment (mabuaisha) [inline registration/tests/test_views.py]: @waseem-omar  No this violate the following PEP8 standard  `E125 continuation line with same indent as next logical line`
If we add to the same line we will break the  rules of limit line length

File: registration/tests/test_views.py (hunk 2 above)
Comment (mabuaisha) [inline registration/tests/test_views.py]: @waseem-omar this is as the above one

File: registration/tests/test_views.py (hunk 1 above)
Comment (mabuaisha) [inline registration/tests/test_views.py]: @waseem-omar I changed this by following the `hanging indent` for the above one

**Pull request merged**